html-report/postcss.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  parser: 'postcss-scss',
+  plugins: [
+    require('postcss-import')({ path: ['src'] }),
+    require('postcss-sass-colors'),
+    require('postcss-mixins'),
+    require('precss'),
+    require('postcss-inline-svg'),
+    require('autoprefixer')({
+      browsers: [
+        '> 5%',
+        'last 2 versions',
+        'IE 11'
+      ]
+    }),
+    require('cssnano')({
+      preset: ['default', {
+        discardComments: {
+          removeAll: true,
+        },
+      }]
+    })
+  ]
+}
```

Comment:
Please enable `Ensure line feed at the end of file on Save` in WebStorm so it'll automatically add empty line at the end of file :)
<img width="1014" alt="screen shot 2017-06-07 at 00 43 14" src="https://user-images.githubusercontent.com/967132/26853398-6e3a8ea2-4b1a-11e7-8b96-9c53f880dfb6.png">